src/border_agent/border_agent.hpp

```diff
@@ -144,10 +144,24 @@ class BorderAgent : private NonCopyable
      */
     void HandleMdnsState(Mdns::Publisher::State aState);
 
+#if OTBR_ENABLE_FEATURE_FLAGS
+    /**
+     * Apply the feature flag values to Border Agent features.
+     *
+     * @param[in] aFeatureFlagList  The feature flag list to be applied to OT/OTBR.
```

Comment:
```suggestion
     * @param[in] aFeatureFlagList  A const reference to the feature flag list.
```

Reply:
remove this part and note with TODO:refactor